README.md

```diff
@@ -39,6 +39,7 @@ Some rules are raising issues on unused values ([`no-useless-increment`]), which
 * The output of functions that don't return anything should not be used ([`no-use-of-empty-return-value`]) ([`requires type-check`])
 * Values should not be uselessly incremented ([`no-useless-increment`])
 * Types without members, 'any' and 'never' should not be used in type intersections ([`no-useless-intersection`])
+* Return of value with type Promise should not go without await when it's in try block ([`no-return-promise-inside-try-block`]) ([`requires type-check`])
```

Comment:
we've changed the scope of the rule to consider not only `return` so I suggest to rename rule to `no-try-with-promise`